src/DssSpell.t.sol

```diff
@@ -222,81 +37,77 @@ contract DssSpellTest is DssSpellTestBase {
         checkCollateralValues(afterSpell);
     }
 
-    function testAAVEDirectBarChange() public {
-        DirectDepositLike join = DirectDepositLike(addr.addr("MCD_JOIN_DIRECT_AAVEV2_DAI"));
-        assertEq(join.bar(), 3.5 * 10**27 / 100);
+    address constant WALLET1 = 0x3C32F2ca11D92a7093d1F237161C1fB692F6a8eA;
+    address constant WALLET2 = 0x2BC5fFc5De1a83a9e4cDDfA138bAEd516D70414b;
+    function testPayments() public {
```

Comment:
I think `vat.sin` and `vat.can` checks are missing, following previous pattern for direct dai payments.

Reply:
Yeah I think we can drop those. They were added from previous spells for extra safety and the vat.can was due to a vat permission being added in a spell which is no longer the case.

Reply:
yeah, as discussed in chat we could generalize the direct dai payment test function and move it in base tests for future use in specif tests, in a separate pr eventually.